assets/dev/js/modules/imports/args-object.js

```diff
@@ -1,4 +1,32 @@
 export default class ArgsObject {
+	static [Symbol.hasInstance]( obj ) {
```

Comment:
Can I have a better explanation of this method and what is it used for?

Reply:
Without this, we can lose functionally of `instanceOf`

without it this is will not work:
```javascript
mycommand = new class Command extends $e.modules.document.CommandHistory {
						getHistory() {}
					}( {} )

console.log( mycommand instanceof $e.modules.CommandBase ) 
// result: false
```

in future we planning to validate commands, if the are instanceOf  CommandBase.

Reply:
@iNewLegend please JS Doc explanation

Reply:
> @iNewLegend please JS Doc explanation

already done, in line 3

Reply:
It looks like since there are no anonymous classes anymore, this functionality is now redundant

Reply:
it's not related to anonymous classes.
please read the JS doc explanation.